### PR TITLE
Added comments, and have quest in brew.bash

### PR DESCRIPTION
**Pull request by DenisZackharov**
После запуска начальной версии (до изменений) у меня постоянно в Google Chrome выходит из всех сервисов, не нашел этой фичи в этом репо, главная проблема похоже находится в Brew.bash не понял пока в чем там дело

### CODE REVIEW

File: modules/functions.bash [hunk 1]
```diff
@@ -1,6 +1,6 @@
-export HOMEBREW_CASK_OPTS="--appdir=/Applications"
-export RBENV_ROOT=/usr/local/var/rbenv
-export NVM_DIR=/usr/local/var/nvm
+#export HOMEBREW_CASK_OPTS="--appdir=/Applications"
```
Comment (DenisZackharov) [inline modules/functions.bash:1]: Тут случайно наверное закоментил

File: Brewfile [hunk 2]
```diff
@@ -5,10 +5,10 @@
 #  RBENV_ROOT=/usr/local/var/rbenv
 #  NVM_DIR=/usr/local/var/nvm
 
-tap "heroku/brew"
+tap "heroku/homebrew-brew" # не понятно для чего это
```
[the remark this answers is not in the document]
Comment (DmitryBarskov) [reply Brewfile:8]: This line adds a third-party repository with heroku-cli

File: modules/osx_defaults.bash [hunk 3]
```diff
@@ -34,7 +34,7 @@ done
 ###############################################################################
 
 # Disable “natural” (Lion-style) scrolling
-defaults write NSGlobalDomain com.apple.swipescrolldirection -bool false
+# defaults write NSGlobalDomain com.apple.swipescrolldirection -bool false # мне кажется это не удобно
```
Comment (DmitryBarskov) [inline modules/osx_defaults.bash:37]: I use "natural" scrolling

File: Brewfile [hunk 4]
```diff
@@ -30,10 +30,10 @@ cask "graphiql"
 cask "iterm2"
 cask "java"
 cask "postman"
-cask "skype"
+#cask "skype"
 cask "slack"
-cask "spectacle"
-cask "sublime-text"
+#cask "spectacle"
+#cask "sublime-text"
```
Comment (DmitryBarskov) [inline Brewfile:36]: Why?

Comment (DenisZackharov) [reply Brewfile:36]: Сейчас не используем вроде

File: modules/osx_defaults.bash (hunk 3 above)
Comment (DenisZackharov) [reply modules/osx_defaults.bash:37]: Тогда убираю получается?

File: Brewfile (hunk 4 above)
Comment (DmitryBarskov) [reply Brewfile:36]: A lot of people use sublime. But yes, I see no people use spectacle

File: modules/osx_defaults.bash (hunk 3 above)
Comment (DmitryBarskov) [reply modules/osx_defaults.bash:37]: Yes, maybe it's better to let the user set this setting himself.

File: Brewfile (hunk 4 above)
Comment (DenisZackharov) [reply Brewfile:36]: okey, sublime will remain)

File: modules/osx_defaults.bash (hunk 3 above)
Comment (DenisZackharov) [reply modules/osx_defaults.bash:37]: I think so too

File: Brewfile (hunk 4 above)
Comment (dmitrytrager) [reply Brewfile:36]: I see VS code cask below. Do we suggest having two editors now?

File: modules/functions.bash (hunk 1 above)
Comment (dmitrytrager) [reply modules/functions.bash:1]: Should we uncomment it then?

File: modules/brew.bash [hunk 5]
```diff
@@ -14,8 +14,8 @@ fi
 info_echo "Install Brew formalue"
 
 brew tap "Homebrew/bundle" 2> /dev/null
-brew bundle install --file="$osx_bootstrap/Brewfile"
+brew bundle install --file="$osx_bootstrap/Brewfile" # ошибка здесь command not found
```
Comment (dmitrytrager) [inline modules/brew.bash:17]: Looks like you need to investigate the root cause of errors

File: modules/osx_defaults.bash [hunk 6]
```diff
@@ -131,12 +131,11 @@ defaults write com.apple.dock showhidden -bool true
 defaults write com.apple.dock magnification -boolean false
 
 # Show dock on right
-defaults write com.apple.dock orientation -string 'right'
-
+defaults write com.apple.dock orientation -string 'right' # точно ли это удобно?
+ 
 ###############################################################################
-# Safari & WebKit                                                             #
+# Safari & WebKit    # Может сделать такое же для google chrome?                                                          #
```
Comment (dmitrytrager) [inline modules/osx_defaults.bash:137]: I believe you either suggest changes for Chrome or write no comments like this

File: modules/functions.bash (hunk 1 above)
Comment (DenisZackharov) [reply modules/functions.bash:1]: I think so

File: Brewfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (DenisZackharov) [reply Brewfile:8]: Sorry I'm still a bad developer I'll fix it)

File: Brewfile (hunk 4 above)
Comment (DenisZackharov) [reply Brewfile:36]: Will the consequences of having 2 IDEs be serious?
If so suggest which ide is more used sublime or vscode

**Pull request closed without merging**